starfish/image/_stack.py

```diff
@@ -960,3 +961,91 @@ def from_numpy_array(cls, array: np.ndarray) -> "ImageStack":
                     empty.set_slice({Indices.HYB: h, Indices.CH: c, Indices.Z: z}, view)
 
         return empty
+
+    def to_pixel_intensities(self, crop: Tuple[int, int, int]=(0, 0, 0)) -> IntensityTable:
+        """Generate an IntensityTable from all the pixels in the ImageStack
+
+        Parameters
+        ----------
+        crop : Tuple[int, int, int]
+            number of pixels from the image borders (z, y, x) to ignore
+
+        Returns
+        -------
+
+        """
+        zmin, ymin, xmin = crop
+        zmax = self.shape['z'] - zmin
+        ymax = self.shape['y'] - ymin
+        xmax = self.shape['x'] - xmin
+        data = self.numpy_array.transpose(2, 3, 4, 1, 0)  # (z, y, x, ch, hyb)
+        cropped_data = data[zmin:zmax, ymin:ymax, xmin:xmax, :, :]
+        intensity_data = cropped_data.reshape(-1, self.num_chs, self.num_hybs)  # (pixels, ch, hyb)
+        z = np.arange(zmin, zmax)
```

Comment:
nit: i think it's better to set up *min, *max, and x/y/z before mucking around with the data.

Reply:
Added a comment to make it clear that these are indices based on the mucked around data.